mods/bones/init.lua

```diff
@@ -5,8 +5,22 @@
 
 -- Load support for MT game translation.
 local S = minetest.get_translator("bones")
-
-bones = {}
+-- bones are supposed to hold up to 4*8+6+4*3*8+4+3*3 item slots:
```

Comment:
This is MTG. No external-mod-specific calculations here. As said, make this variable.